sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -53,14 +53,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/martygrant/unified-runtime.git")
   # commit 036b9cfcd8fb4be9394449b5406e6402580a63c9
   # Merge: 2ab0734 86f96f0
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
   # Date:   Fri Oct 27 16:36:23 2023 +0100
   #     Merge pull request #961 from hdelan/change-unions-to-stdvariant
   #     [HIP][CUDA] Change unions in ur_mem_handle_t_ to stdvariant
-  set(UNIFIED_RUNTIME_TAG 036b9cfcd8fb4be9394449b5406e6402580a63c9)
+  set(UNIFIED_RUNTIME_TAG bd362a12379bdd66fb4b26f60a709df7a33a6092)
```

Now that https://github.com/oneapi-src/unified-runtime/pull/968 is merged please pull in the latest sycl branch and apply this patch before making the PR ready for review.

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit ba994bb401aefad3631012d592be672f6ea0e23b
  # Merge: be53fb3b f2a365ca
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Fri Nov 3 11:09:52 2023 +0000
  #     Merge pull request #968 from martygrant/martin/versionCheckSpecializationConstants
  #     [OpenCL] Add version check for urProgramSetSpecializationConstants
```

---

Updated thanks.